src/components/SEO/index.js

```diff
@@ -26,14 +26,18 @@ function SEO({ description, lang, meta, title }) {
   );
 
   const metaDescription = description || site.siteMetadata.description;
+  let pageTitle;
```

Comment:
```suggestion
  let pageTitle = `${title} · ${site.siteMetadata.title}`;
  if (homepage) {
    pageTitle = `${site.siteMetadata.title} · ${title}`;
  }
```

And remove the `} else { [...] }` below.

Reply:
Groovy, I was thinking that might be a better approach.